packages/entities/entities-routes/vite.config.ts

```diff
@@ -20,6 +20,7 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
     rollupOptions: {
       external: [
         '@kong-ui-public/expressions', // This is optional if we do not use Expressions features
+        '@kong-ui-public/expressions/dist/style.css', // This is optional if we do not use Expressions features
```

Comment:
Does this mean we need to import this style file on the host app side?

Reply:
I’m not sure I understand not bundling the style here. If they are scoped they’ll be excluded.

Reply:
> if they do, they should install expressions manually

So how would this be handled in KM or Konnect?

Reply:
They installed both of expressions and entities-route
https://github.com/Kong/konnect-ui-apps/pull/3745
https://github.com/Kong/kong-manager/pull/256